patches/third_party-widevine-cdm-widevine.gni.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/third_party/widevine/cdm/widevine.gni b/third_party/widevine/cdm/widevine.gni
+index 58f073ca562ca40c3165f4dd0f650dc467b43d5c..8668af33c10b49df982ec4eb92cfaefe9302feff 100644
+--- a/third_party/widevine/cdm/widevine.gni
++++ b/third_party/widevine/cdm/widevine.gni
+@@ -20,10 +20,10 @@ declare_args() {
+ if (is_chromeos && !is_chromeos_device) {
+   enable_widevine = false
+ }
+-
```

Comment:
?

Reply:
Resolved in 07e9aeebff86e2f79ba0b39aa06bdb6291f39659.